cli/src/main/package/setup.bat

```diff
@@ -36,6 +36,7 @@ if "%USER_PATH%" == "" (
 ) else (
   setx PATH "%USER_PATH%;%CD%\bin"
 )
+cd ..
 setx IDE_ROOT "%CD%"
```

Comment:
You also need to return to the original directory (`_ide`) after the `setx` command since we later run the `setup` bash script that will otherwise not be found anymore and will fail.